doc/api/test.md

```diff
@@ -2240,6 +2240,47 @@ test('mocks a builtin module in both module systems', async (t) => {
 });
 ```
 
+### `mock.value(object, valueName, value)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `object` {Object} The object whose value is being mocked.
+* `valueName` {string|symbol} The identifier of the value on object to mock.
+* `value` {any} A value used as the mock value for `object[valueName]`.
```

Comment:
The return value is missing here. See the docs for `mock.method()` for an example.